portal/patient/fwk/libs/verysimple/Phreeze/Phreezer.php

```diff
@@ -527,47 +527,52 @@ public function Get($objectclass, $id, $cache_timeout = null)
         if (is_null($cache_timeout)) {
             $cache_timeout = $this->ObjectCacheTimeout;
         }
-        
+
         if (strlen($objectclass) < 1) {
             throw new Exception("\$objectclass argument is required");
         }
 
         if (strlen($id) < 1) {
             throw new Exception("\$id argument is required for $objectclass");
         }
-        
+
         // see if this object was cached & if so return it
         $obj = $cache_timeout == 0 ? null : $this->GetCache($objectclass, $id);
         if ($obj) {
             return $obj;
         }
-        
+
         $pkm = $this->GetPrimaryKeyMap($objectclass);
-        
+
         if (! $pkm) {
             throw new Exception("Table for '$objectclass' has no primary key");
         }
-        
-        $criteria = new Criteria();
-        $criteria->PrimaryKeyField = "`" . $pkm->TableName . "`.`" . $pkm->ColumnName . "`";
-        $criteria->PrimaryKeyValue = $id;
-        
+        if ($objectclass === 'Patient') {
```

Comment:
Do the endpoints that use this ever get called from within core openemr (in the openemr portal dashboard)?

Reply:
This needs to be conditional on if portal or not. I'll push a fix